### DISCUSSION TITLE
Setup manual without installer.lua

### DISCUSSION THREAD

**Discussion by amsitlab** [lumen-oss/rocks.nvim#207] [Q&A] [answered]
I have luarocks 3.11.0 installed on my machine
Can I setup rocks.nvim manully without installer.lua ?

**Comment by mrcjkb** [comment, marked as answer]
Yes, you can. [Here's a tutorial](https://github.com/nvim-neorocks/rocks.nvim/wiki/Installing-rocks.nvim-manually,-without-the-installation-script) :smile: 

If neovim can find luarocks on the PATH, you can also tell the installer script to use your existing luarocks installation.
You just need to make sure luarocks is configured to use lua 5.1.
